docs/pages/foundation/spacing.html

```diff
@@ -101,7 +101,7 @@ <h3 class="docs-header-h3">Sections</h3>
     </div>
 </div>
 
-<img src="{{site.baseurl}}/images/spacing/sections.png" alt="section spacing" class="img-responsive">
+<img src="{{site.baseurl}}/images/spacing/sections.png" alt="section spacing" width="100%">
```

Comment:
@saad-mo ~ I don't know if we want to use attributes for styling. What was the reasoning for moving away from the class?

Thoughts @xak?

Reply:
`img-responsive` was a bootstrap class

Reply:
@saad-mo ~ Right on! If you could, let's make our own class that does something similar. We want to avoid inline styles wherever possible, especially as HTML attributes.

Reply:
cool. added `img-fliud` utility class for docs and updated markup.